tutorial/4_hextof_workflow.ipynb

```diff
@@ -347,6 +600,13 @@
     "sp.add_energy_offset()"
    ]
   },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## plot the delayStage values"
```

Comment:
This does only output noise for me. Probably something goes wrong with energy calibration/offsetting, but I don't have a clue. 
In any case, if you run this from the tutorials folder, it will pick up parameters from the sed_config.yaml file, which is there by default, so either we remove this, or disable the folder_config in any processor call here. This is, however, not the reason for this energy calibration problem, becaue it persists if I remove the file.

Reply:
I removed the sed_config.yaml, as it does not belong there. Also this has never been a problem for me as the work directory is my home, not that folder. 
I cannot reproduce the problem you mention, my plotting works just fine. It would be good if you could identify the source of the problem, as it is likely in the functionalities of the calibration/corrections.

Reply:
There is also an other problem which I still have not figured out, and that is why the energy calibration adds the correct offset when calculated, and when applied after saving the parameters, all is offset by 2eV

Reply:
Seems to work now, cannot reproduce the problem. The issue with the 2 eV I think come from the the fact that in the one case you offset for the mean of your bias serias, which spans about 4 eV, so it is about 2 eV smaller than the higher bias, which I think was used for the latter experiment. So for that case, you experience that 2 eV offst. 
This is what I want to get rid off by taking the actial bias voltage into account in the energy calibration, and subtracting it there already. Then you can add the full value of the bias, and don't have to subtract the mean in the shift function, which produces the problem in the first place.

Reply:
> Seems to work now, cannot reproduce the problem. The issue with the 2 eV I think come from the the fact that in the one case you offset for the mean of your bias serias, which spans about 4 eV, so it is about 2 eV smaller than the higher bias, which I think was used for the latter experiment. So for that case, you experience that 2 eV offst.

Does this mean that setting ref_id to reference the bias which will be used later will set the correct energy calibration parameters for that ref_id?

> This is what I want to get rid off by taking the actial bias voltage into account in the energy calibration 

This would be great. We could even add an arbitrary number of columns, which then, when adding them as offset, we don't need to compensate the mean.